dictionaries/CallMap.php

```diff
@@ -12043,11 +12043,11 @@
 'SessionHandler::validateId' => ['bool', 'session_id'=>'string'],
 'SessionHandler::write' => ['bool', 'id'=>'string', 'data'=>'string'],
 'SessionHandlerInterface::close' => ['bool'],
-'SessionHandlerInterface::destroy' => ['bool', 'session_id'=>'string'],
-'SessionHandlerInterface::gc' => ['bool', 'maxlifetime'=>'int'],
-'SessionHandlerInterface::open' => ['bool', 'save_path'=>'string', 'name'=>'string'],
-'SessionHandlerInterface::read' => ['string', 'session_id'=>'string'],
-'SessionHandlerInterface::write' => ['bool', 'session_id'=>'string', 'session_data'=>'string'],
+'SessionHandlerInterface::destroy' => ['bool', 'id'=>'string'],
+'SessionHandlerInterface::gc' => ['int|bool', 'max_lifetime'=>'int'],
```

Comment:
```suggestion
'SessionHandlerInterface::gc' => ['int|false', 'max_lifetime'=>'int'],
```
Ref: https://www.php.net/manual/en/sessionhandlerinterface.gc.php
> Returns the number of deleted sessions on success, or `false` on failure.

Reply:
Not sure about this one. The text is interpreted as `int|false`, but the signature itself as presented in the docs uses `int|bool`. Which one should be the deciding one? I guess one could provide a change to the PHP docs for this particular one?

Reply:
Source code is the ultimate truth, and it says `MAY_BE_LONG|MAY_BE_FALSE`: https://github.com/php/php-src/blob/787db3af484a4680b2aa25bd1dcc6a9b09f2c1f2/ext/session/session_arginfo.h#L113-L115

PHP docs have been pretty inaccurate historically, specifically for arg/return types. This situation is improving, but is far from perfect.

Reply:
👍  Will push the change.